python/ray/internal/internal_api.py

```diff
@@ -4,8 +4,26 @@
 
 import ray.worker
 from ray import profiling
+import pyarrow
 
-__all__ = ["free"]
+__all__ = ["free", "pin_object_data"]
+
+
+def pin_object_data(obj_id):
+    """Pin the object data referenced by this object id in memory.
```

Comment:
nice! is there an unpin? 

And does this move out from the Tune repo?

And is this exposed on ray.readthedocs.io?

Reply:
realistic to just call `free` instead of unpin?